### DISCUSSION TITLE
Order parameters output method

### DISCUSSION THREAD

**Discussion by Wolke926** [idaholab/moose#25586] [Q&A Modules: Phase field]
Dear MOOSE users,

Are there any methods we can use for order parameters' value output in a 2D simulation？ If I just open the Paraview output file, these different values are only identified in colors, although we have a scale bar, I am still wondering if there are more straightforward ways.

**Comment by dschwen** [comment]
Could you clarify what you want the output to look like? Are we talking about visualizing a grain structure? Or do you simply want to map the output space? You could use a ParsedAux kernel to compute new values from a combination of order parameter values, using any expression you'd like.
